cmd/config-migrate.go

```diff
@@ -2409,29 +2415,65 @@ func migrateV27ToV28() error {
 	return nil
 }
 
-// Migrates '.minio.sys/config.json' v27 to v28.
+func migrateV28ToV29() error {
```

Comment:
We don't need to do this anymore. We only need to migrate the backend config..

Reply:
Done